mitiq/interface/mitiq_pennylane/tests/test_conversions_pennylane.py

```diff
@@ -104,29 +104,27 @@ def test_non_consecutive_wires_error():
 
 
 def test_integration():
-    n_wires = 4
-
     gates = [
-        qml.PauliX,
-        qml.PauliY,
-        qml.PauliZ,
-        qml.S,
-        qml.T,
-        qml.RX,
-        qml.RY,
-        qml.RZ,
-        qml.Hadamard,
-        qml.Rot,
-        qml.CRot,
-        qml.Toffoli,
-        qml.SWAP,
-        qml.CSWAP,
-        qml.U1,
-        qml.U2,
-        qml.U3,
-        qml.CRX,
-        qml.CRY,
-        qml.CRZ,
+        qml.PauliX(wires=0),
+        qml.PauliY(wires=0),
+        qml.PauliZ(wires=0),
+        qml.S(wires=0),
+        qml.T(wires=0),
+        qml.RX(0.4, wires=0),
+        qml.RY(0.4, wires=0),
+        qml.RZ(0.4, wires=0),
+        qml.Hadamard(wires=0),
+        qml.Rot(0.4, 0.5, 0.6, wires=1),
+        qml.CRot(0.4, 0.5, 0.6, wires=(0, 1)),
+        qml.Toffoli(wires=(0, 1, 2)),
+        qml.SWAP(wires=(0, 1)),
+        qml.CSWAP(wires=(0, 1, 2)),
+        qml.U1(0.4, wires=0),
+        qml.U2(0.4, 0.5, wires=0),
+        qml.U3(0.4, 0.5, 0.6, wires=0),
+        qml.CRX(0, wires=(0, 1)),
```

Comment:
```suggestion
        qml.CRX(0.4, wires=(0, 1)),
```
To avoid an operation equivalent to the identity.

Reply:
Done.